src/assembler.cpp

```diff
@@ -1242,6 +1242,9 @@ int main(int argc, char* argv[])
                         constant.b = (uint32_t)values[0] != 0;
                         break;
                     }
+
+                    default:
```

Comment:
I'd rather have the warning be kept around than making the default case explicit by guess.